SSH tunnel connection: PostgreSQL and Redshift (#1163)
rudderlabs/rudderstack-docs: docs/destinations/warehouse-destinations/redshift.mdx

- **User**: The name of the user with the required read/write access to the above database.
- **Password**: The password for the above user.
- **Namespace**: Enter the schema name where RudderStack will create all the tables. If you don't specify any namespace, RudderStack will set this to the source name, by default.

### SSH connection

<div class="infoBlock">
This feature is available only for the <a href="https://www.rudderstack.com/pricing/">Growth and Enterprise plan</a> users.
</div>

RudderStack lets you connect to your Redshift database securely over an SSH connection. Enter the following details to do the same:

- **SSH Connection**: Toggle on this boolean setting to use the SSH connection while connecting to your Redshift database.
- **SSH Host**: Enter the IP address of your bastion host.
- **SSH Port**: Enter the port for the above host.
- **SSH User**: Enter the user name of your Redshift database.

> - **SSH Public Key**: Copy the public key provided in this field and add it to the `authorizated_keys` file on your bastion host. Rudderstack will use the private key corresponding to this public key to establish the connection successully.

Reviewer: Should we also show snapshot of how the screen looks when we capture the ssh credentials ?

Author: Added an image without the credentials though. We will need to blur those even if we capture. LMK what do you think.

Reviewer 2: Same question as above - should it be `authorized_keys`? just verifying if this isn't a typo.

- **User**: The name of the user with the required read/write access to the above database.
- **Password**: The password for the above user.
- **Namespace**: Enter the schema name where RudderStack will create all the tables. If you don't specify any namespace, RudderStack will set this to the source name, by default.

### SSH connection

<div class="infoBlock">
This feature is available only for the <a href="https://www.rudderstack.com/pricing/">Growth and Enterprise plan</a> users.
</div>

> SSH tunneling is a method of transporting data over an encrypted SSH connection. You can use it to add encryption to your legacy applications as well as achieve compliance such as HIPAA, PCI-DSS, and other standards without having the need to modify the existing applications.

Reviewer 2: ```suggestion
SSH tunneling is a method of transporting data over an encrypted SSH connection. You can use it to add encryption to your legacy applications and achieve compliance like HIPAA, PCI-DSS, and other standards, without having to modify the existing applications.
```

> SSH tunneling is a method of transferring data over an encrypted SSH connection. You can use it to add encryption to your legacy applications as well as achieve compliance such as HIPAA, PCI-DSS, and other standards without having the need to modify the existing applications.

Reviewer 2: ```suggestion
SSH tunneling is a method of transferring data over an encrypted SSH connection. You can use it to add encryption to your legacy applications and achieve compliance with regulations like HIPAA, PCI-DSS, etc., without having to modify the existing applications.
```

> RudderStack lets you connect to your Redshift database securely over an SSH connection. Enter the following details to do the same:

Reviewer 2: ```suggestion
RudderStack lets you connect to your Redshift database securely over an SSH connection by configuring these settings:
```

<img src="../../assets/warehouse-destinations/ssh-connection.png" alt="Redshift edit configuration" />

> - **SSH Connection**: Toggle on this boolean setting to use the SSH connection while connecting to your Redshift database.

Reviewer 2: ```suggestion
- **SSH Connection**: Enable this setting to use the SSH connection while connecting to your Redshift database.
```

- **SSH Host**: Enter the IP address of your bastion host.
- **SSH Port**: Enter the port for the above host.

> - **SSH User**: Enter the user name of your Redshift database.

Reviewer: `Enter the username used to access the bastion host`

- **SSH Public Key**: Copy the public key provided in this field and add it to the `authorizated_keys` file on your bastion host. Rudderstack will use the private key corresponding to this public key to establish the connection successully.

> You can also enable the SSH connection for an existing Amazon Redshift destination by navigating to the destination's **Configuration** tab, selecting **Edit configuration** and enabling the **SSH connection** setting.

Reviewer 2: @ishachopra0401 same change as in the Postgres doc if we agree on the structure.

> You can also enable the SSH connection for an existing Amazon Redshift destination by navigating to the destination's **Configuration** tab, selecting **Edit configuration** and enabling the **SSH connection** setting.

Reviewer 2: ```suggestion
To enable the SSH connection for an existing Redshift destination, navigate to the destination's **Configuration** tab, select **Edit configuration** and enable the **SSH connection** setting.
```
